plugins/modules/cloudwatch_metric_alarm_info.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or https://wwww.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: cloudwatch_metric_alarm_info
+version_added: 5.0.0
+short_description: Gather information about the alarms for the specified metric
+description:
+  - Retrieves the alarms for the specified metric.
+author:
+  - Mandar Vijay Kulkarni (@mandar242)
+options:
+  alarm_names:
+    description:
+      - The name of the metric.
+    required: false
+    type: list
+    elements: str
+  alarm_name_prefix:
+    description:
+      - An alarm name prefix to retrieve information about alarms that have names that start with this prefix.
+      - Can not be used with I(alarm_names).
+    required: false
+    type: str
+  alarm_type:
+    description:
+      - Specify this to return metric alarms or composite alarms.
+      - Module is defaulted to return metric alarms but can return composite alarms if I(alarm_type=composite_alarm).
+    required: false
+    type: str
+    default: metric_alarm
+    choices: [composite_alarm, metric_alarm]
```

Comment:
I guess it should be `['CompositeAlarm', 'MetricAlarm']` https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudwatch.html#CloudWatch.Client.describe_alarms